db/migrate/20151213185304_add_slug_to_page.rb

```diff
@@ -0,0 +1,6 @@
+class AddSlugToPage < ActiveRecord::Migration
+  def change
+    add_column :pages, :slug, :string, null: false
```

Comment:
Here is what I propose (as can be read in the `db/migrate/20151213185304_add_slug_to_page.rb` file):

```
add_column :pages, :slug, :string, null: false, default: 'temporary-slug'

# NOTE: we'll need to add a unique index on that 'slug' column after:
# 1) the current migration has been run in production
# 2) all slugs from pages with a 'temporary-slug' slug have been changed

# The unique index on the 'slug' column should look like this:
# add_index :pages, :slug, unique: true
```